std/StandardToken.sol

```diff
@@ -21,11 +21,11 @@ contract StandardToken is Token {
 		return supply;
 	}
 
-	function transfer(address _to, uint256 _value) returns (bool success) {
```

Comment:
I can code that up, but it feels like a real kludge that is avoiding the issue.  The issue here is that return variables are getting used for two different things.  The first is that they are used to create a variable you can assign a value to.  The second is that they are strings which document the meaning of the return value.  The compiler is very much written with the first use in mind.  This is to say that the parser does not make a very strong distinction between an output parameter and any other locally declared variable.

So, I feel like if you're going to create a variable to document your output's meaning, you should probably also have to use the variable.  I've gone ahead and made a change here to actually use `success` rather than remove it.  It doesn't feel much harder to assign your output parameters and let the return happen automatically than to do a simple `return`.

If you don't like the way it looks, I'll look into "return with value" counting as a use of the return variables.